README.md

```diff
@@ -103,12 +104,6 @@ NixOS modules ([src](modules/modules.nix))
   * [backups](modules/backups.nix): duplicity backups of all your node's important files
   * [operator](modules/operator.nix): configures a non-root user who has access to client tools (e.g. `bitcoin-cli`, `lightning-cli`)
 
-### Extension modules
-Extension modules are maintained in separate repositories and have their own review
-and release process.
-
-* [Mempool](https://github.com/fort-nix/nix-bitcoin-mempool): Bitcoin visualizer, explorer and API service
-
```

Comment:
Do you think it makes sense to keep the extension module section and maybe link to an archived version of the mempool module?

Reply:
No. What would be the benefit of this?

Reply:
To document that extension modules are possible and how to create them.

Reply:
I've added `nix-bitcoin-mempool` to the examples.